docs/usage_tips/permissions.md

```
@@ -0,0 +1,151 @@
+
+Users can access all plugin pages, refresh data from LibreNMS, and review comparison tables, but cannot import devices or sync data.
+
+### Full Plugin Access
+
+- **Plugin permissions**: View + Change (Can view LibreNMS Plugin Pages and Import and sync devices data)
+- **NetBox permissions**: Add/change permissions for devices, interfaces, cables, IP addresses
+
+Users have full access to all plugin features and can import devices, sync interfaces, and create cables.
+
+
+## Further Details
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)